### PR TITLE
2.13 upgrades

### PR DESCRIPTION
**Pull request by Dhaulagiri**

### CODE REVIEW

File: config/targets.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* eslint-env node */
+
+module.exports = {
+  browsers: [
```
Comment (locks) [inline config/targets.js:4]: is this now the default?

Comment (Dhaulagiri) [reply config/targets.js:4]: it's what `ember init` gives you but beyond that I don't know since we aren't actually using this in any of our apps

Comment (Dhaulagiri) [reply config/targets.js:4]: for the purposes of learning it might be easier to not include this to avoid introducing a potentially confusing concept?

Comment (acorncom) [reply config/targets.js:4]: If it's part of `ember init`, I'd vote for leaving it in.  We don't explain everything about Ember during the tutorial and this is probably something to skip ...

**Pull request merged**